specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -835,6 +835,55 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/addNodeUser": {
```

Comment:
>}/ [](start = 162, length = 2)

how do we remove user from the node ?
Why not model this as a new user resource (proxy extension)  ?.. it will allow template deployments and allow deletes more naturally.